notes/notes.bib

```diff
@@ -7,3 +7,29 @@ @article{bubeck
   number = {3-4},
   pages = {231-357},
 }
+
+@inproceedings{Duchi2008,
+address = {New York, New York, USA},
+author = {Duchi, John and Shalev-Shwartz, Shai and Singer, Yoram and Chandra, Tushar},
+booktitle = {Proceedings of the 25th international conference on Machine learning - ICML '08},
+doi = {10.1145/1390156.1390191},
+isbn = {9781605582054},
+pages = {272--279},
+publisher = {ACM Press},
+title = {{Efficient projections onto the l 1 -ball for learning in high dimensions}},
+url = {http://portal.acm.org/citation.cfm?doid=1390156.1390191},
+year = {2008}
+}
+
+@article{Bentbib2015,
```

Comment:
This is not the right reference. It's probably best to leave it out. I may add a more appropriate reference later.